pilot/pkg/dns/monitoring.go

```diff
@@ -30,14 +30,14 @@ var (
 	)
 
 	failures = monitoring.NewSum(
-		"dns_total_upstream_failures",
+		"dns_upstream_failures_total",
 		"Total number of DNS requests forwarded to upstream.",
 	)
 
 	requestDuration = monitoring.NewDistribution(
```

Comment:
We need to change the .Record call to use seconds now I think

Reply:
correct. Missed that. Thanks for catching it.